packages/wrangler/src/pages/publish.tsx

```diff
@@ -389,3 +419,29 @@ export const Handler = async ({
 	);
 	await metrics.sendMetricsEvent("create pages deployment");
 };
+
+/**
+ * Custom _routes.json files need to run through optimization
+ * to potentially reduce the overall worker pipeline size
+ */
+function validateAndOptimizeRoutesFile(
```

Comment:
Lets change this to `validateRoutesFile()` and remove the optimization - we have a ticket to stop optimizing custom `_routes.json` files